common/src/main/java/com/linecorp/centraldogma/internal/Jackson.java

```diff
@@ -74,8 +77,16 @@
         compactMapper.disable(SerializationFeature.INDENT_OUTPUT);
         prettyMapper.enable(SerializationFeature.INDENT_OUTPUT);
         // Sort the attributes when serialized via the mapper.
-        compactMapper.enable(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS);
-        prettyMapper.enable(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS);
+        enable(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS);
+
+        // Enable JSON5 features.
+        enable(JsonReadFeature.ALLOW_SINGLE_QUOTES,
```

Comment:
I think we shouldn't use these options for `JSON`.
We need to add `JacksonJson5` as we do in https://github.com/line/centraldogma/pull/616/files#diff-202d69d7f9ebcf08990eda99e556947155f88acc967601e20a3df8361c4aef6bR28

Reply:
Indeed, I tried the YAML PR's approach(`Jackson.ofJson()`, `Jackson.ofJson5()`) which I think a bit verbose.
We actually need only a few methods of what `Jackson` currently have to support `YAML` and `JSON5` while `Jackson` class is currently being used as a general utility for json ser/deser.

So what about having separate utility classes for `YAML` and `JSON5` and use it instead?
We can then save lots of meaningless code modifications.

Reply:
> We actually need only a few methods of what Jackson currently have to support YAML and JSON5 while Jackson class is currently being used as a general utility for json ser/deser.

Hmm, could you tell me more about the "few methods" part, please?
Because IIRC, there are quite a lot of methods that take `String` and `byte[]`, which should exist in the shape of JSON5.
We should use different mappers for JSON/JSON5, otherwise, we are going to loosen the validation.

Reply:
> > Because IIRC, there are quite a lot of methods that take String and byte[], which should exist in the shape of JSON5.

Right, there're quite a lot of read methods but `readTree` method is the only method used for `JSON5` validation and `JsonNode` conversion in this PR.

Reply:
Hmm, I might misunderstad something. 😅 
Let's see one of the methods:
```java
public static <T> T readValue(String data, Class<T> type) {
    try {
        return compactMapper.readValue(data, type);
    ...
}
```
Let's say that we set JSON5 configuration to the `compactMapper` and use it to read JSON/JSON5 data together.
When users read JSON data, which should throw a `JsonParseException` because it's invalid, it might not throw the exception because it's a valid JSON5 type. I think this is unexpected behavior.

Or did you mean specifying a different `mapper`? 🤔

Reply:
> Or did you mean specifying a different mapper? 🤔

Yeah, since what I only needed was `readTree(...)` method with JSON5 capabilities, I thought we can leave `Jackson` class as it is and make another utility class for `JSON5`(with JSON5 enabled mapper) whose only method is `readTree(...)` and use it where we need to read `JSON5` string.

Reply:
Had a chat with @ks-yim and we agreed that there's no need to introduce `Jaskson.json5()`.
It's an internal class and there are not so many methods needed for JSON5 so introducing `Json5` will be enough. 😄